pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -1697,6 +1697,20 @@ status:
       type: Succeeded
 `)
 	)
+	featureFlags, _ := config.NewFeatureFlagsFromMap(map[string]string{
+		"enable-api-fields": "alpha",
+	})
+
+	failedOnReconcileFailureTaskRun.Status.Provenance = &v1beta1.Provenance{
+		FeatureFlags: featureFlags,
+	}
+	failedOnReconcileFailureTaskRun.Status.RetriesStatus[0].Provenance = &v1beta1.Provenance{
+		FeatureFlags: featureFlags,
+	}
+
+	retriedTaskRun.Status.Provenance = &v1beta1.Provenance{
+		FeatureFlags: config.DefaultFeatureFlags.DeepCopy(),
+	}
```

Comment:
why not change the yamls?

Reply:
Done. Adding it into yamls directly. PTAL. Thank you!